src/lib/components/side-nav.svelte

```diff
@@ -27,15 +25,5 @@
       />
     {/if}
   {/each}
-  <svelte:fragment slot="bottom">
-    <NavigationButton
-      onClick={() => ($useDarkMode = !$useDarkMode)}
-      tooltip={$useDarkMode
-        ? translate('common.night')
-        : translate('common.day')}
-      label={$useDarkMode ? translate('common.night') : translate('common.day')}
-      icon={$useDarkMode ? 'moon' : 'sun'}
-    />
-    <slot name="bottom" />
-  </svelte:fragment>
+  <slot name="bottom" />
```

Comment:
I think we'll need the below. This is definitely tricky and will help when we move to Svelte5 (don't need that for this PR). Basically there's multiple slots that we need to pass down through.

`<svelte:fragment slot="bottom" name="bottom" >`

or 

```
<svelte:fragment slot="bottom">
  <slot name="bottom">
</svelte:fragment>